lms/static/js/groups/views/cohort_editor.js

```diff
@@ -44,9 +44,11 @@ var edx = edx || {};
                 tabName = tabElement.data('tab');
             event.preventDefault();
             this.$('.wrapper-tabs .tab').removeClass('is-selected');
+            this.$('.wrapper-tabs .tab').find('span.sr').remove();
             tabElement.addClass('is-selected');
+            tabElement.find('a').prepend('<span class="sr">Selected tab: </span>');
```

Comment:
I thought we had started to internationalize screen reader comments, though it may not be required. Check with @talbs